NullableTypeCases.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HelloWorld
+{
+    public class TaxiDriver
+    {
+        public int SleepHours;
+
+
+        public TaxiDriver() { }
+      
+    }
+
+    public static class TaxiDriverExtensions
+    {
+        public static void CompareAndStoreBiggerValue(ref int sleepHours, int? workHours)
+        {
+            if (Nullable.Compare<int>(sleepHours, workHours) < 0)
```

Comment:
great implementation. just want to make sure if you fully understand it. e.g. if you could re-write it in simpler way?